app/views/feeds/_form_apple_config.html.erb

```diff
@@ -6,38 +6,66 @@
     <div class="card-body" data-controller="apple-key">
       <div class="mb-4">
         <p><%= t(".description") %></p>
-        <a href="https://help.prx.org/hc/en-us/articles/26602803552155-Setting-up-Apple-Podcast-Subscriptions-with-Apple-Delegated-Delivery"><%= t(".guide_link") %></a>
+        <%= t(".guide_link").html_safe %>
```

Comment:
this is a cosmetic change- help urls should not be embedded in the code, moved it to the translations, and allowed a link with `html_safe`